Gemfile

```diff
@@ -13,9 +13,6 @@ gem 'yard', '~> 0.8.7', :require => false
 group :documentation do
   gem 'redcarpet',     '2.3.0'
   gem 'github-markup', '1.0.0'
-  if RUBY_VERSION > '2.0.0'
-    gem 'relish'
-  end
```

Comment:
Shouldn't this be added without the gate?

Reply:
That was a bad delete ;)